catalogue/webapp/components/SearchForm/SearchForm.js

```diff
@@ -40,7 +40,9 @@ const ClearSearch = styled.button`
 `;
 
 const SearchForm = ({ ariaDescribedBy, compact }: Props) => {
-  const { query, workType } = useContext(CatalogueSearchContext);
+  const { query, workType, _queryType, setQueryType } = useContext(
```

Comment:
Why the underscore for `_queryType`?

Reply:
The API has decided that "secret" queries are underscored, thought we might as well not deviate.